### PR TITLE
Customize admin interface for UserProfile

### PR DESCRIPTION
**Pull request by anuschka**
Addressed #145.

### CODE REVIEW

**Comment by mojcas** [comment]
@anuschka please consider updates in UserProfile made in PR #167 as there some fields were removed, for example UserProfile 'user', 'fullname', 'githubuser', since these fields are already part of Django User model, and are being duplicated unnecessarily in UserProfile model.

**Comment by gandalfar** [comment]
Ok, lets figure out the order in which to merge these two PR's.

**Comment by anuschka** [comment]
I can easily close mine and create it again.

**Comment by anuschka** [comment]
But can we really remove these fields from the UserProfile model? When I look into the admin I can see only User, Provider, UID and Extradata provide from Github authentication which we use....

File: TrackCat/settings.py [hunk 1]
```diff
@@ -56,6 +56,8 @@
 	'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 	'django.contrib.messages.middleware.MessageMiddleware',
 	'django.middleware.clickjacking.XFrameOptionsMiddleware',
+	'django.contrib.auth.middleware.AuthenticationMiddleware',
+	'django.contrib.messages.middleware.MessageMiddleware',
```
Comment (gandalfar) [inline TrackCat/settings.py]: You have these 2 midllewares already in this list. You don't need them twice (so just remove them).

File: api/admin.py [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+		(None,				{'fields': ['user', 'fullname', 'githubuser', 'email','website']}),
+        ('Biography',		{'fields': ['bio']}),
+        ('Activity',		{'fields': ['active','activity_status']}),
+    ]
+	list_display = ('user', 'fullname', 'githubuser', 'activity_status')
+	list_filter = ('activity_status',)
+
 admin.site.register(Task)
-admin.site.register(UserProfile)
+admin.site.register(UserProfile,UserProfileAdmin)
```
Comment (gandalfar) [inline api/admin.py]: space after comma

**Comment by mojcas** [comment]
UserProfile as per relations, is kind of an "extension" of User (Django object), which has already what we need... for example, it has: username, first_name, last_name, email.. I think we can see it here:
https://docs.djangoproject.com/en/1.7/ref/contrib/auth/

**Comment by gandalfar** [comment]
That's not a problem, updating admin.py can be part of refactoring of UserProfile. I'll merge this and you can move the conversation to the Issue.

**Pull request merged**